src/main/webapp/app/shared/user-settings/ssh-settings/ssh-user-settings.component.html

```diff
@@ -3,80 +3,153 @@ <h1 jhiTranslate="artemisApp.userSettings.sshSettings">
 </h1>
 @if (currentUser) {
     <div class="list-group d-block">
-        <div class="list-group-item">
-            <dt>
-                <span jhiTranslate="artemisApp.userSettings.sshSettingsPage.sshSettingsInfoText"> </span>
-                <jhi-documentation-button [type]="documentationType" />
-            </dt>
-        </div>
-        @if (storedSshKey === '' && !editSshKey) {
-            <div class="list-group-item">
-                <jhi-button
-                    [btnType]="ButtonType.PRIMARY"
-                    [btnSize]="ButtonSize.SMALL"
-                    [title]="'artemisApp.userSettings.sshSettingsPage.addNewSshKey'"
-                    (onClick)="editSshKey = !editSshKey"
-                />
-            </div>
-        }
-        @if (storedSshKey !== '' && !editSshKey) {
-            <dt class="list-group-item" jhiTranslate="artemisApp.userSettings.sshSettingsPage.sshKeyDisplayedInformation"></dt>
-            <div class="list-group-item">
-                {{ sshKey }}
-            </div>
-            <div class="list-group-item">
-                <div class="btn-group" role="group" aria-label="Actions">
+        <!-- Initial state: There are no keys -->
+        @if (keyCount === 0 && !showSshKey) {
+            <div class="list-group-item px-4 py-4">
+                <div class="mt-4"></div>
+                <h4 class="text-center mb-4 mt-8" jhiTranslate="artemisApp.userSettings.sshSettingsPage.noKeysHaveBeenAdded">No SSH keys have been added</h4>
+                <div class="text-center mt-4 d-flex justify-content-center align-items-center flex-wrap" style="max-width: 500px; margin: 0 auto">
+                    <p>
+                        <span style="font-size: medium" jhiTranslate="artemisApp.userSettings.sshSettingsPage.whatToUseSSHForInfo"> </span>
```

Comment:
I would generally prefer css classes over inline styles with more than one element (like `max-width: 500px; margin: 0 auto`) or those you use multiple times (like `font-size: medium`)